modules/data/schematics/ng-add/index.ts

```diff
@@ -260,6 +293,8 @@ export default function(options: EntityDataOptions): Rule {
     const parsedPath = parseName(options.path, '');
     options.path = parsedPath.path;
 
+    const templateSource = createEntityConfigFile(options, parsedPath.path);
```

Comment:
I would rather stay consistent with the rest of the schematic and "inline" this as:

```ts
options.entityConfig
  ? createEntityConfigFile(options, parsedPath.path)
  : noop();
```